### DISCUSSION TITLE
[Error] 无法部署 workers

### DISCUSSION THREAD

**Discussion by jipiyan** [dreamhunter2333/cloudflare_temp_email#474] [Q&A]
<img width="1379" alt="image" src="https://github.com/user-attachments/assets/1fbeb4f5-08a4-4d6d-8d42-6694778ff36b">

下载了 release 页面中的 0.7.6 版本 worker.js。按照教程点开左侧文件浏览，删除旧文件，上传了新文件后，点击部署。但是在页面下方出现了 no module 错误，请大佬帮忙解惑一下

**Comment by dreamhunter2333** [comment]
可以先回退到上个版本，我测试下

**Comment by dreamhunter2333** [comment]
CF 更新了 UI 部署需要手动在设置修改此处，我后续增加到文档

<img width="1046" alt="image" src="https://github.com/user-attachments/assets/5e6e7c96-d11b-4efe-87a8-e1e6b1f93ff2">

**Comment by dreamhunter2333** [comment]
UI 部署需要手动在设置修改此处，已加到文档
https://temp-mail-docs.awsl.uk/zh/guide/ui/worker.html
